src/game.cpp

```diff
@@ -8098,10 +8182,13 @@ void game::examine( const tripoint &examp )
         Creature *c = critter_at(examp);
         monster *mon = dynamic_cast<monster *>(c);
 
-        if( mon != nullptr && mon->has_effect("pet") ) {
+        if( mon != nullptr && mon->has_effect("pet") && !mon->has_effect("wary") ) {
```

Comment:
Unselectable.
Some note why don't they work would be good, but I'm not sure where to put that without bloating the code too much.
In the critters name in pet menu maybe? Something like "Tentacle dog (wary of you)".